woocommerce/woocommerce-ios
# Store creation M3: update category question options with the full list of industries from web & Stripe

Part of #8376 


## Description


For the store category profiler question, we want to match the options with web p1672465997982299/1672316116.056879-slack-C045CUK1Y3U with the source of truth in [this json](https://github.com/Automattic/woocommerce.com/blob/trunk/themes/woo/start/config/options.json). Since there are 158 categories, I wrote some script in Swift Playground to generate the code for a new enum `Category` and its display name from the web json. In the updated design HyVloP5FipZzyPVenH2euI-fi-3906%3A138822&t=7pkrx9VC2FH2gaKX-0, we're also showing categories (industries) by the group based on the [Stripe doc](https://support.stripe.com/questions/setting-an-industry-group-when-creating-a-stripe-account). Most of the diffs are just enum cases for the 158 categories 😅 

## Testing instructions


- Log in if needed
- Go to the Menu tab, and tap `Switch store`
- On the store picker, tap `+ Add a store`
- Tap `Create a new store` --> the store name form should be shown
- Enter a store name and continue --> the category question should be shown with a `Skip` CTA at the top and a `Continue` CTA at the bottom, with a list of categories separated into sections with a header
- Scroll down the list to reach the end to make sure everything looks as expected
- Select an option and tap `Continue` --> the domain selector should be shown
- Select a domain and continue --> the selected category name should be displayed on the store summary below the URL

## Screenshots


dark | light
-- | --
![Simulator Screen Shot - iPhone 14 Pro Max - 2023-01-05 at 14 13 37](https://user-images.githubusercontent.com/1945542/210715844-519aab4f-2481-4757-a1ed-a0702dcb2289.png) | ![Simulator Screen Shot - iPhone 14 Pro Max - 2023-01-05 at 14 14 12](https://user-images.githubusercontent.com/1945542/210715858-cf09c4a8-76b7-4099-8748-18c67529470a.png)


---
